<?xml version="1.0"?>
<launch>
    
<!-- SLAM地図作成ノード -->

    <param name="use_sim_time" value="false"/>
    <node pkg="gmapping2" type="slam_gmapping" name="slam_gmapping" clear_params="true">
      <rosparam command="load" file="$(find pitakuru)/config/gmapping.yaml" />
      <remap from="scan" to="/scan_main" />
      <param name="throttle_scans" value= "1"/>
      <param name="map_frame" value="map"/>
      <param name="base_frame" value="base_link" />
      <param name="odom_frame" value="odom" />
    </node>

</launch>
